rep-2000.rst

```diff
@@ -306,6 +307,109 @@ Build System Support:
+Binary releases are provided as a single archive per platform containing all packages in [this repos file](https://github.com/ros2/ros2/blob/master/ros2.repos).
+
+" [s] " Compilation from source, the ROS buildfarm will not produce any binary packages for these platforms.
+
+Middleware Implementation Support:
+
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| Middleware Library       | Middleware Provider | Support Level | Platforms                   | Architectures                        |
++==========================+=====================+===============+=============================+======================================+
+|  rmw_fastrtps_cpp        | eProsima Fast-RTPS  | Tier 1        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+|  rmw_connext_cpp         | RTI Connext         | Tier 1        | All Platforms except Debian | All Architectures except arm64/arm32 |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_opensplice_cpp       | ADLink OpenSplice   | Tier 2        | All Platforms except Debian | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+| rmw_fastrtps_dynamic_cpp | eProsima Fast-RTPS  | Tier 2        | All Platforms               | All Architectures                    |
++--------------------------+---------------------+---------------+-----------------------------+--------------------------------------+
+
+Middleware implementation support is dependent upon the platform support tier.
+For example a Tier 1 middleware implementation on a Tier 2 platform can only receive Tier 2 support.
+
+Minumum language requirements:
+
+- C++14
+- Python 3.5
+
+
+Dependency Requirements:
+
++-------------+-------------------------------------------------+---------------------+
+|             |                        Required Support         | Recommended Support |
++-------------+----------------+---------------+----------------+---------------------+
+| Package     | Ubuntu  Bionic |     MacOS**   |   Windows 10** | Debian Stretch [s]  |
++=============+================+===============+================+=====================+
+| CMake       |     3.10.2     |     3.12.2    |     3.12.1     |        3.7.2        |
++-------------+----------------+---------------+----------------+---------------------+
+| EmPY        |     3.3.2      |     3.3.2     |     3.3.2      |        3.3.2        |
++-------------+----------------+---------------+----------------+---------------------+
+| Ogre        |                                      1.10*                            |
```

Comment:
Just curious, why are the ogre versions not listed for Ubuntu and Mac? (they were listed for Bouncy but not Crystal)

Reply:
The cell containing the value `1.10*` spans across all columns.